### PR TITLE
v1.4.0_SSR-2116_MobileDatePicker_component

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

**Review by trung-doan — APPROVED** [review]

**Review by trung-doan — APPROVED** [review]

File: src/base/datetime/mobile-date/test/inputId.test.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      const container = document.createElement("kuc-mobile-base-date");
+      const el = await fixture(container);
+      const inputEl = el.querySelector(
+        ".kuc-mobile-base-date__group__input"
+      ) as HTMLInputElement;
+
+      expect(inputEl.id).to.equal("-label");
+    });
+
+    it("should 'sample-id-lable' when assigned ", async () => {
```
Comment (nmanhit) [inline src/base/datetime/mobile-date/test/inputId.test.ts:16]: 'sample-id-lable' >> 'sample-id-label'

File: src/base/datetime/mobile-date/test/inputAriaInvalid.test.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { expect, fixture, elementUpdated } from "@open-wc/testing";
+import "../index";
+
+describe("BaseMobileDate", () => {
+  describe("inputAriaInvalid", () => {
+    it("should false when not assigned ", async () => {
```
Comment (nmanhit) [inline src/base/datetime/mobile-date/test/inputAriaInvalid.test.ts:6]: remove white space at end of line

File: src/base/datetime/mobile-date/test/inputAriaInvalid.test.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      const container = document.createElement("kuc-mobile-base-date");
+      const el = await fixture(container);
+      const inputEl = el.querySelector(
+        ".kuc-mobile-base-date__group__input"
+      ) as HTMLInputElement;
+
+      expect(inputEl.getAttribute("aria-invalid")).to.equal("false");
+    });
+
+    it("should true when assigned ", async () => {
```
Comment (nmanhit) [inline src/base/datetime/mobile-date/test/inputAriaInvalid.test.ts:16]: emove white space at end of line

File: src/base/datetime/mobile-date/test/inputId.test.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { expect, fixture, elementUpdated } from "@open-wc/testing";
+import "../index";
+
+describe("BaseMobileDate", () => {
+  describe("inputId", () => {
+    it("should '-label' when not assigned ", async () => {
```
Comment (nmanhit) [inline src/base/datetime/mobile-date/test/inputId.test.ts:6]: emove white space at end of line

**Pull request merged**